inst/stan/epinowcast.stan

```diff
@@ -118,6 +118,9 @@ data {
   int rep_rncol; 
   matrix[rep_fncol, rep_rncol + 1] rep_rdesign; 
   array[2, 1] real rep_beta_sd_p;
+  // Probability aggregation indicators
+  int agg_probs;
```

Comment:
we should probably assign this a prefix like repagg_probs and co to match the others?